### PR TITLE
Fix find help text formatting.

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cmd/find-main.go [hunk 1]
```diff
@@ -97,47 +97,52 @@ USAGE:
...
+UNITS
+   --smaller, --larger flags accept human-readable case-insensitive number suffixes
+   such as "k", "m", "g" and "t" referring to the metric units KB, MB, GB and TB
+   respectively. Adding an "i" to these prefixes, uses the IEC units, so that "gi"
+   refers to "gibibyte" or "GiB". A "b" at the end is also accepted. Without suffixes
+   the unit is bytes.
+
+   --older, --newer flags accept the suffixes "d", "w", "m" and "y" to refer to
+   units of days, weeks, months and years respectively. With the standard rate
+   of conversion being 7 days being in 1 week, 30 days in 1 month, and 365 days
```
Comment (poornas) [inline cmd/find-main.go]: s/being 7 days being in 1 week/being 7 days in 1 week/

Comment (harshavardhana) [reply cmd/find-main.go]: Done

Comment (poornas) [reply cmd/find-main.go]: The --older and --newer flags are not recognized. Throws a parse error
➜  mc git:(e9f0357b) ✗ mc find play/uploads   
play/uploads/moon-horizon-720p.jpg
➜  mc git:(e9f0357b) ✗ mc find play/uploads --smaller 1GB
play/uploads/moon-horizon-720p.jpg
➜  mc git:(e9f0357b) ✗ mc find play/uploads --larger 1GB 
➜  mc git:(e9f0357b) ✗ mc find play/uploads --older 1w  
mc: <ERROR> Error parsing string passed to flag older strconv.Atoi: parsing "": invalid syntax

Comment (harshavardhana) [reply cmd/find-main.go]: Oh we might need to fix that, i wonder what broke we tested this before @CarterMcClellan .

Comment (harshavardhana) [reply cmd/find-main.go]: We will have to fix this outside of this PR @poornas

**Comment by harshavardhana** [comment]
More changes done @poornas can you take a look?

**Pull request merged**